mono/metadata/mono-basic-block.c

```diff
@@ -517,6 +531,95 @@ mono_basic_block_free (MonoSimpleBasicBlock *bb)
 	}
 }
 
+
+typedef struct {
+	int index;
+	int loop_index;
+	GPtrArray *stack;
+} scc_data;
```

Comment:
A comment what "scc" means?

Reply:
strongly connected component, a pretty common abbreviation in graph theory